src/error.rs

```diff
@@ -25,8 +26,8 @@ pub enum Error {
     #[fail(display = "Base64 Decode error: {}", _0)]
     Decode(#[fail(cause)] base64::DecodeError),
 
-    #[fail(display = "RNG error: {}", _0)]
-    Rng(#[fail(cause)] rand::Error),
+    #[fail(display = "Crypto error: {}", _0)]
+    Crypto(#[fail(cause)] CryptoError),
```

Comment:
This might warrant mention in the upgrade notes, too?

Reply:
Yes, this is a backwards incompatible change. I'll write up a set of changes on Monday.